module/Application/view/layout/layout.phtml

```diff
@@ -21,7 +21,12 @@
                                 'href' => $this->basePath() . '/img/favicon.ico'
                             ])
             ->prependStylesheet($this->basePath('css/style.css'))
-            ->prependStylesheet($this->basePath('css/bootstrap.min.css'))
+            ->prependStylesheet('https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css',
```

Comment:
Can we use a [version range](https://www.jsdelivr.com/documentation#id-use-a-version-range-instead-of-an-exact-version-only-works-with-valid-semver-versions) here?

Reply:
What would be the version range? All of v5 or all of v5.3?

Reply:
I think it would be safer to just use a 5.3 version range.  

However, it would require some work on our end to upgrade to 5.4 and later versions, but not that much.

Reply:
I updated to use 5.3